source/includes/steps-atlas-terraform-file-from-template.yaml

```diff
@@ -0,0 +1,242 @@
+---
+title: "In |vscode-short|, create a new file."
+ref: create-file-vscode
+level: 4
+content: |
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 33 33 34
+
+     * - Operating System
+       - Method
+       - Actions
+
+     * - Any
+       - |vscode-short| Menu
+       - :guilabel:`File` > :guilabel:`New File`
+
+     * - MacOS
+       - Keyboard Shortcut
+       - Press ``Command`` + ``N``
+
+     * - Windows and Linux
+       - Keyboard Shortcut
+       - Press ``Control`` + ``N``
+
+---
+title: "Save the file to a new directory. Name the file ``main.tf``"
+ref: change-language
+level: 4
+content: |
+
+  .. note::
+
+     ``main.tf`` is the recommended filename for the entry point to a
+     Terraform module.
+
+---
+title: "In the ``main.tf`` file, type ``atlas``, then press the **Enter** or **Return** key."
+ref: bootstrap-atlas
+level: 4
+content: |
+
+  This populates the ``main.tf`` file with an example configuration 
+  using the `MongoDB Atlas
+  <https://www.terraform.io/docs/providers/mongodbatlas/index.html>`__ 
+  Terraform provider to create a Shared Tier |service| cluster. 
+
+  Update the example configuration to configure your cluster.
+
+---
+title: "Update the |service| Terraform configuration to configure your cluster."
+level: 4
+ref: update-config
+content: |
+
+  The value of the ``name`` argument of the ``mongodbatlas_project``
+  resource named ``my_project`` gains focus.
+
+  Provide values for the following arguments to configure your cluster:
+
+  .. tip::
+
+     Press the **Tab** key to change the focus to the next argument in 
+     the template that you should update.
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 33 33 34
```

Comment:
FYI these don't have to add up to 100 - they're purely ratios. So you could accomplish 3 equal columns with any of the following:

- `:widths: 33 33 33`
- `:widths: 20 20 20`
- `:widths: 1 1 1`

Reply:
didn't know that. thanks for the tip! updated.